fleet_packages.json

```diff
@@ -15,22 +15,22 @@
 [
   {
     "name": "apm",
-    "version": "8.0.0"
+    "version": "8.1.0"
   },
   {
     "name": "elastic_agent",
     "version": "1.3.0"
   },
   {
     "name": "endpoint",
-    "version": "1.4.1"
+    "version": "1.5.0"
   },
   {
     "name": "fleet_server",
     "version": "1.1.0"
   },
   {
     "name": "synthetics",
-    "version": "0.9.0"
+    "version": "0.9.2"
```

Comment:
@dominiqueclarke - it looks like `synthetics-0.9.2` is no longer available in the snapshot EPR. Looking at the [latest build](https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fpackage-storage/detail/snapshot/2198/pipeline/295) I only see `0.9.0` listed in the build step. Do you know if it was removed intentionally? 

Other links I checked
- https://epr-snapshot.elastic.co/search?package=synthetics&experimental=true
- https://epr-snapshot.elastic.co/package/synthetics/0.9.2/?experimental=true

cc @joshdover - this looks to be the cause for this backport failing. A build in the interim between your PR being green and this backport removed this synthetics version from snapshot.

Reply:
hmm, I'm not sure that this is the case after looking further. 0.9.2 is present in the latest build further down:

<img width="1192" alt="image" src="https://user-images.githubusercontent.com/1965714/155228687-a11a2372-c9ff-45e1-b23c-566605a4f7cb.png">

(observation: when I visited https://epr-snapshot.elastic.co/search?package=synthetics&experimental=true for the first time, I only saw 0.9.0, but after refreshing it updated to 0.9.2. caching is fun.)

regardless, the tests here should be locked down to this docker image anyway: `docker.elastic.co/package-registry/distribution@sha256:8b4ce36ecdf86e6cfdf781d9df8d564a014add9afc9aec21cf2c5a68ff82d3ab`
running this image locally does show 0.9.2 present

I noticed that 8.1 branch of these tests do not contain `experimental: true`:
https://github.com/elastic/kibana/blob/43953d81d9a1276c6c88713b38e7740bc145d2e3/x-pack/test/functional_synthetics/services/uptime/synthetics_package.ts#L52-L53
compared to main branch:
https://github.com/elastic/kibana/blob/ccc9488f93d4817dcbecd81947be10cff1663b6e/x-pack/test/functional_synthetics/services/uptime/synthetics_package.ts#L52-L53

going to update this PR with that experimental flag. though how these tests ever ran without the flag is a mystery to me (AFAIK, synthetics package only have `beta` versions)